.github/workflows/integration-tests.yml

```diff
@@ -78,6 +78,8 @@ jobs:
               git checkout ceadf6ea68c5ef316abfc618879f2cf9290e45b3
               composer install
               ../../../phpstan.phar analyse
+              # test phpstan/4147 (restricted open_basedir) and phpstan-src/1451 (disabled proc_open)
+              php -d open_basedir="$(pwd):/tmp" -d disable_functions="pcntl_exec,pcntl_fork,exec,passthru,proc_open,shell_exec,system,popen" ../../../phpstan.phar analyse
```

Comment:
Separate matrix item please

Reply:
You don't need to test it with Laravel, just create a new directory with a small "project" specific to this test.

Reply:
No because of two things:
- larger project is expected to use at least one preload using composer `autoload/files` (https://github.com/laravel/framework/blob/ceadf6ea68c5ef316abfc618879f2cf9290e45b3/composer.json#L106)
- and because only a larger project will generate multiple parallel jobs.

for ex. the https://github.com/slevomat/coding-standard.git is already analysed twice in `.github/workflows/integration-tests.yml` /w different configurations

with this reasoning, is it ok to keep the 2nd reanalyse of Lavarel here too?

Reply:
I don't want to misuse Laravel for this, I'm testing Laravel for a different reason. The assumption might change for a different reason.

just create a new directory with a small "project" specific to this test.

You can force multiple child processes for a smaller amount of files if you tweak these settings https://phpstan.org/config-reference#parallel-processing.

Reply:
Done, all feedback addressed. https://github.com/phpstan/phpstan-src/pull/1466 must be merged first.